test/feature-benchmark/mzcompose.py

```diff
@@ -363,3 +364,128 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             f"ERROR: The following scenarios have regressions: {', '.join([scenario.__name__ for scenario in scenarios.keys()])}"
         )
         sys.exit(1)
+
+
+def workflow_mzcloud(c: Composition, parser: WorkflowArgumentParser) -> None:
+    parser.add_argument(
```

Comment:
This code is duplicated with the default workflow. If you prefer, I can refactor in a follow-up PR.